modules/ROOT/pages/provisioning-hyperv.adoc

```diff
@@ -0,0 +1,104 @@
+
+image::new_hyperv.png[HyperV New Virtual Machine Wizard]
+
+This will launch the _New Virtual Machine Wizard_.  You can follow the guided wizard but take note of the following two exceptions.
+
+==== Configuring networking
+
+When asked about the networking in the _New Virtual Machine Wizard_, be sure set up networking or your virtual machine will not be able to communicate with anything.  It is sufficient to select the _Default_ Networking option.
+
+==== Configuring the Fedora CoreOS image as the disk
+You need to assign the HDX disk image you downloaded earlier as the virtual machine's primary disk. This is not the default setting.  You must click _Use an existing virtual disk_ and then specify the location of the image you downloaded earlier.  See the image below for an example.
+
+image::hyperv-disk.png[]
+
+=== Setting the Ignition config
+
+Once your virtual machine is defined, do not start it.  Be sure to complete the following section before starting your virtual machine. When first booting Fedora CoreOS, you need to have an ignition file which is used to set up and configure the operating system.
+
+For the HyperV platform, Ignition will read its configuration file from the HyperV hypervisor in parts and then reassemble it.  Before this can occur, you can use `kvpctl` and its _add-ign_ subcommand to split your Ignition file and assign it to the virtual machine.  The syntax for the command is as follows:
+
+[source, powershell]
+----
+.\kvpctl.exe <name_of_vm> add-ign <path_to_ign_file> <ignition_config_prefix>
+----
+
+When Ignition is run on first boot, it will look for a pre-defined key name and reassemble the Ignition configuration file.  The pre-defined key name is *ignition.config.*.  Be sure to note the trailing `.` on the prefix key name.
+
+For example:
+
+[source, powershell]
+----
+.\kvpctl.exe podman-machine-default add-ign C:\Users\joe\myvm.ign ignition.config.
+added key:  ignition.config.0
+added key:  ignition.config.1
+added key:  ignition.config.2
+added key:  ignition.config.3
+added key:  ignition.config.4
+added key:  ignition.config.5
+added key:  ignition.config.6
+----
+
+
+=== Starting the VM
+
+Before starting the VM, be certain you have completed the step xref:_setting_the_ignition_config[ Setting the Ignition config].
+
+There are several ways to start a virtual machine depending on the context of the HyperV Manager graphical interface.  To start a virtual machine, for example,  you can right-click on the virtual machine and select _Start_.
+
+=== Viewing the key value pairs assigned to your virtual machine
+
+You can view the key value pairs that are assigned to your machine with `kvpctl` and its _get_ subcommand. You may only get key value pairs when the virtual machine is _running_.  This is by design from Microsoft.
+
+[source, powershell]
+----
+.\kvpctl.exe <name_of_vm> get
+----
+
+For example:
+
+[source, bash]
+----
+> .\kvpctl.exe myvm get
```

Comment:
```suggestion
[source, powershell]
----
.\kvpctl.exe myvm get
```

According to the [docs](https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-highlighter/#shell-vs-console) a `source` block shouldn't include the prompt if declared as a specific shell language.

Also the example should probably be based around Powershell to facilitate using Windows native tooling (as implied by the `.exe`).

Reply:
This example shows output from the command, so `console` seems like the right choice.